### PR TITLE
Removes Dorsisdwarf

### PR DESCRIPTION
**Pull request by Dorsisdwarf**
REMOVE DORSISDWARF remove dorsisdwarf
you are worst dwarf. you are the dwarf idiot you are the dwarf smell. return to croatioa. to our coder cousins you may come our contry. you may live in the zoo….ahahahaha ,dorsisdwarf we will never forgeve you. cetnik rascal FUck but fuck asshole dwarf stink dorsisdwarf sqhipere shqipare..dwarf genocide best day of my life. take a bath of dead dwarf..ahahahahah

### CODE REVIEW

**Comment by Xhuis** [comment]
:older_man: :gun: :man:
dorsisdwarf removal squad here

**Comment by xxalpha** [comment]
:skull:

**Comment by duncathan** [comment]
meme pr :^)

**Comment by MrStonedOne** [comment]
Holy fucking diff! What did you do to your line endings

File: config/admins.txt [hunk 1]
```diff
@@ -91,4 +91,8 @@ drovidicorv = Game Master
 Dunc = Game Master
 MMMiracles = Game Master
 bear1ake = Game Master
-CoreOverload = Game Master
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (Iamgoofball) [inline config/admins.txt]: >merge conflicts in the admin txt

top kek/10

File: html/changelogs/example.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 #################################
 
 # Your name.  
-author: N3X15
+author: Dorsidwarf
```
Comment (Iamgoofball) [inline html/changelogs/example.yml]: >EDITING THE EXAMPLE CHANGELOG

File: config/tips.txt [hunk 3]
```diff
@@ -1,156 +1,156 @@
...
-As a Gangster, your influence is based on how many areas you have tagged and how many people are wearing your gang's outfit; more areas and more people wearing the outfit will give you more influence.
-As a Gang Boss, it is wise to promote at least one other person you think is good or effective, preferably two, as a backup if you get caught by security or enemy gangsters.
-Gang outfits are very robust, giving moderate resistances to most direct damage at the cost of stealth.
-As an Abductor, you can select where your victims will be sent on the ship control console.
-As an Abductor Agent, the combat mode vest has much higher resistance to every kind of weapon, and your helmet prevents the AI from tracking you.
-As an Abductor, the baton is set up to allow you to rapidly stun, sleep, and cuff: From stun mode, it cyles to sleep, then to cuff, allowing you to render a target helpless and unable to communicate very rapidly.
-As a Ghost, you can see the inside of a container on the ground by clicking on it.
+
+Dragging yourself or someone else onto a chair or bed will buckle them to it, immobilizing them. If handcuffed, it will take time to resist out.
```
Comment (Iamgoofball) [inline config/tips.txt]: >BREAKING LINE ENDINGS IN EVERY FUCKING TXT FILE

**Comment by Dorsisdwarf** [comment]
You are the worst smell coder committing old branch . genocide of dwarf is happiest day of life ... go die fire dwarf dorsi

**Comment by bgobandit** [comment]
memes

**Comment by Aranclanos** [comment]
t-thanks for your contribution

**Comment by Miauw62** [comment]
wtf

**Pull request closed without merging**